docs/getstarted/fleet/groups.mdx

```diff
@@ -0,0 +1,44 @@
+# Device groups
+
+Artemis allows you to organize your devices in groups. Each group has an
+associated remote pod reference. This allows Artemis to determine which
+pod a device is supposed to run.
+
+Device groups are defined in the `fleet.json` file:
+
+```
+{
+  ...
+  "groups": {
+    "wifi": {
```

Comment:
I think it's easier to follow if the name contains "-group". Otherwise, readers might assume that these groups already exist.
Alternatively, using group names that don't have any meaning otherwise (like "office", "home") could also make sense.

```suggestion
    "wifi-group": {
```

Maybe we should add a note that we use "-group" in the documentation, but that it's custom not to include that suffix.